config/shared/errors/errors.go

```diff
@@ -67,8 +69,10 @@ var (
 	ErrNoPath                    = errors.New("path not specified")
 	ErrPathRelative              = errors.New("path not absolute")
 	ErrDirtyPath                 = errors.New("path is not fully simplified")
-	ErrSparesUnsupportedForLevel = errors.New("spares unsupported for arrays with a level greater than 0")
+	ErrRaidLevelRequired         = errors.New("raid level is required")
+	ErrSparesUnsupportedForLevel = errors.New("spares unsupported for linear and raid0 arrays")
 	ErrUnrecognizedRaidLevel     = errors.New("unrecognized raid level")
+	ErrRaidDevicesRequired       = errors.New("raid devices is required")
```

Comment:
`is` is intentional, but I'm open to feedback.

Reply:
I'm not opposed to it, if you're looking for alternatives maybe  `raid devices required`?